### ISSUE TITLE
react-example page links cannot be accessed directly

### ISSUE THREAD

**Issue by twschiller** [pixiebrix/playground#20]
For example: https://pixiebrix.github.io/playground/react-example/table

Also, on the table page, we have a couple links for showing off cross-tab communication. These are broken since they link to pages within the application

Related:
- #19

**Comment by twschiller** [comment]
@fregante do you happen to know if GitHub pages supports this? (Or vercel or netlify?)

If we were doing it on a webserver, each example subdirectory would have to serve up it's root index for all subresources/paths

**Comment by fregante** [comment]
> do you happen to know if GitHub pages supports this

Unfortunately not. If we had 1 project per repo then we can use `404.html` as the entry point for the app. Unfortunately GH Pages only reads the one at the root, not `/react-example/404.html` for example.

> Or vercel or netlify

Vercel definitely can do it if you create a project for each folder.

If we don't want to setup a Vercel project for each folder, I think it's still possible via `routes` in Vercel.json. I can try setting it up, if you'd like.

**Comment by twschiller** [comment]
Yes, that'd be great if you could try getting a basic Vercel setup. I should just create a PixieBrix Vercel account and add you?

It looks like routes has been replaced with: https://vercel.com/docs/cli#project-configuration/rewrites

**Comment by fregante** [comment]
I'll test it on my account and then see how it goes. It'll be easier to change settings if I have full access
